boards/arm/ruuvi_ruuvitag/ruuvi_ruuvitag.dts

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2020 Ruuvi Innovations Ltd (Oy)
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52832_qfaa.dtsi>
+
+/ {
+	model = "Ruuvi Ruuvitiag";
+	compatible = "ruuvi,ruuvi-ruuvitag";
```

Comment:
This should be `ruuvi,ruuvitag` (manufacturer,model) since the product is `RuuviTag` (capitalized that way; please check the PR for `Ruuvitag` and fix it).

Reply:
Thanks that has been change now.